server/backup_binlog.go

```diff
@@ -76,20 +76,25 @@ func (s *backupBinlogService) FlushAndBackupBinlog(ctx context.Context, req *age
 		Bucket: aws.String(req.BucketName),
 		Key:    aws.String(req.BackupId),
 	})
+
 	if err == nil {
 		s.agent.sem.Release(1)
 		return nil, status.Errorf(codes.InvalidArgument, "the requested backup has already completed: BackupId=%s", req.BackupId)
 	}
+	metrics.SetBinlogBackupInProgressMetrics(s.agent.clusterName, true)
```

Comment:
This seems too early.
Placing this just before `flushBinaryLogs()` would be appropriate.